src/lib/operators/import.hpp

```diff
@@ -18,7 +15,9 @@ namespace hyrise {
 /*
  * This operator reads a file, creates a table from that input and adds it to the storage manager.
  * Supported file types are .tbl, .csv and Hyrise .bin files.
- * For .csv files, a CSV config is additionally required, which is commonly located in the <filename>.json file.
+ * For .csv files, a CSV config is additionally required (if no table with the same name exists as a template).
+ * The operator looks for this content in the file <filename>.json.
+ * If a table with this filename already exists, it is dropped. This contradicts the COPY FROM semantic from SQL.
```

Comment:
I think Daniel told me that COPY FROM is not SQL standard, but Postgres (@dey4ss, right?).
I would write ".. it is dropped (not appended as done in Postgres)".